packages/block-library/src/post-excerpt/block.json

```diff
@@ -23,7 +23,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
This block has a read more link as part of it: should we add link as one of the defaults?